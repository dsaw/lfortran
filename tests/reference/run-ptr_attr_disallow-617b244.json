{
    "basename": "run-ptr_attr_disallow-617b244",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/./errors/ptr_attr_disallow.f90",
    "infile_hash": "7f051d7957b482c1d810a94655853d46c8c3a5853802453a8d517002",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-ptr_attr_disallow-617b244.stderr",
    "stderr_hash": "f95b0fc9bc7255866a10c665316d9d26f35eeda52364b0871fcefd5a",
    "returncode": 1
}