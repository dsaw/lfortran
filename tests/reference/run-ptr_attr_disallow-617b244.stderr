semantic error: `PARAMETER` attribute conflicts with `POINTER` attribute
 --> tests/./errors/ptr_attr_disallow.f90:3:5
  |
3 |     integer, pointer, parameter :: v => null()
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
